README.md

```diff
@@ -49,33 +25,45 @@ Running from Dockerhub
    git pull origin main
    ```
 
-4. Create a [Mistral AI token](https://console.mistral.ai/api-keys/). 
+4. Build the Docker image using the following command:
 
-   Then create a `.env` file  (Do not commit this file to git) and add the following:
-   ```
-   MISTRAL_API_KEY=<Mistral AI key>
-   ```
+   Befere running this command include API key in the end 
 
-5. Build the Docker image:
+   Go to [team3](https://csusb.instructure.com/courses/43192/discussion_topics/419698) in canvas for API key
+   
    ```
-   docker build -t team3-app .
+   docker build -t team3-app . --build-arg MISTRAL=
    ```
 
-6. Now, run the Docker container:
-   ```
+5. Steps to delete the previous image 
```

Comment:
I think it's better to do this before building docker image, as like a preliminary step.

Reply:
can u please check now